observation_portal/accounts/views.py

```diff
@@ -102,3 +109,48 @@ def get_response_serializer(self, *args, **kwargs):
 
     def get_endpoint_name(self):
         return 'requestAccountRemoval'
+
+
+class BulkCreateUsersApiView(CreateAPIView):
+    """API endpoint to create users in bulk"""
+
+    permission_classes = [IsAdminUser | IsPrincipleInvestigatorOfAnyProposal]
+    serializer_class = BulkCreateUsersSerializer
```

Comment:
This serializer class should be dynamically loaded too in case users want to override it.